docs/docs/explore/ai-chat.md

```diff
@@ -0,0 +1,176 @@
+- **Accuracy** - Charts are generated from the same data sources as your dashboards
+- **Clarity** - Complex data patterns become immediately visible
+
+:::tip From Chat to Dashboard
+Think of AI Chat as your starting point for data exploration. Use it to quickly answer questions and spot trends, then leverage the dashboard links to dive deeper with full filtering, comparison, and export capabilities.
+:::
+
+## Improving AI Chat with Instructions
+
+To get the most accurate and contextual responses from AI Chat, you can add custom `ai_instructions` to your project files. These instructions provide the AI with additional context about your data, business logic, and preferred response formats.
+
+### Why Add AI Instructions?
+
+AI instructions help the AI:
+- Understand your specific business context and terminology
+- Format responses in ways that match your team's preferences
+- Focus on the metrics and dimensions most relevant to your use case
+
+### Where to Add Instructions
+
+You can add `ai_instructions` in two places:
+
+1. **`rill.yaml`** - Project-wide instructions that apply to all queries across your entire project
+2. **`<metrics_view>.yaml`** - Metrics view-specific instructions for individual dashboards
+
+For detailed examples and best practices on writing effective AI instructions, see the [Rill MCP documentation](/explore/mcp#adding-ai-instructions-to-your-model).
+
+## Use in Your Favorite AI Client
+
+Want to chat with your Rill data in Claude Desktop, ChatGPT, or other AI assistants? The **Rill MCP Server** lets you connect your Rill projects to external AI clients, giving you the same powerful, governed analytics experience outside of Rill Cloud.
+
+### Why Use an External Client?
+
+Using the Rill MCP Server with external AI assistants is ideal for:
+- **Integration with workflows** - Combine Rill data queries with other tools and capabilities in your AI client
+- **Power users** - Data teams and analysts who want full control over their AI environment
+
+### Getting Started with Rill MCP Server
+
+To connect your Rill projects to external AI clients like Claude Desktop:
+
+1. **Check the prerequisites** - You'll need Node.js and an MCP-compatible client
+2. **Create a personal access token** - Generate a token for authentication (for Rill Cloud projects)
+3. **Configure your client** - Add the Rill MCP server configuration to your AI client
+4. **Start chatting** - Ask questions about your data using the same predefined metrics
+
+For detailed setup instructions, configuration examples, and troubleshooting, see the **[Rill MCP Server documentation](/explore/mcp)**.
+
+:::info Same Technology, Different Interface
+Both AI Chat and the Rill MCP Server use the same underlying technology, ensuring equally accurate and consistent results based on your metrics views. Choose AI Chat for quick, browser-based queries or MCP Server for integration with your preferred AI assistant.
+:::
```

Comment:
This is too much information about the MCP server here. I think we should just have one paragraph that merely says  (with a bit more color): "Want to chat in your own client? Read our MCP docs to learn more."

Especially the "Getting Started with the Rill MCP Server" section belongs on the MCP docs page itself.